doc/user/layouts/partials/create-source/connector/kafka/with-options.html

```diff
@@ -2,7 +2,7 @@
 `group_id_prefix` | `text` | Use the specified prefix in the consumer group ID. The resulting `group.id` looks like `<group_id_prefix>materialize-X-Y`, where `X` and `Y` are values that allow multiple concurrent Kafka consumers from the same topic.
 `cache` | `boolean` | Cache data from this source to local files. Requires [experimental mode](/cli/#experimental-mode).
 `security_protocol` | `text` | Use [`ssl`](#ssl-with-options) or, for [Kerberos](#kerberized-kafka-details), `sasl_plaintext`, `sasl-scram-sha-256`, or `sasl-sha-512` to connect to the Kafka cluster.
-`statistics_interval_ms` | `int` | `librdkafka` statistics emit interval in `ms`. Accepts values [0, 86400000]. The granularity is 1000ms. A value of 0 disables statistics.
+`statistics_interval_ms` | `int` | `librdkafka` statistics emit interval in `ms`. Accepts values [0, 86400000]. The granularity is 1000ms. A value of 0 disables statistics. Statistics can be queried using the `mz_kafka_consumer_statistics` system table.
```

Comment:
I'm not sure what "The granularity is 1000ms" means -- is it a typo where it should say "The default is 1000ms" or is it saying "whatever number you give us will be rounded to the next greatest second" or something else?

Reply:
I believe, and you'll need to confirm with @elindsey, that this means that the value is rounded to the nearest second (or `floor` or `ceil`, not sure).

Reply:
Also, I didn't add the "granularity" part, it just happens to be on a line that I modified.

Reply:
I know that. I flagged it as weird because I couldn't interpret the what the sentence meant either way - if we end up rounding to second granularity then why are we bothering to take inputs in milliseconds anyway? No pressure if you want to merge without addressing this nit. I'll peek at what the code does and see if I can figure it out in a few min

Reply:
alright this looks like a librdkafka thing and we're just copying their language. 

```
librdkafka statistics emit interval. The application also needs to register a stats callback using rd_kafka_conf_set_stats_cb(). The granularity is 1000ms. A value of 0 disables statistics.
Type: integer
```

from https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md 

feel free to disregard then!